edb/lib/cfg.edgeql

```diff
@@ -178,6 +178,12 @@ ALTER TYPE cfg::AbstractConfig {
             Access-Control-Allow-Origin HTTP header';
     };
 
+    CREATE PROPERTY enable_recompilation -> std::bool {
```

Comment:
The name seems too unspecific.  I'd use something like `auto_rebuild_query_cache`